services/dynamic-scheduler/requirements/_base.in

```diff
@@ -20,4 +20,5 @@ httpx
 packaging
 python-socketio
 typer[all]
+u-msgpack-python
```

Comment:
I did a quick search, and it seems that [msgpack-python](https://github.com/msgpack/msgpack-python) is actively maintained and has 2K stars, while the one you're using had its last release in May 2023 and has only 250 stars.

Reply:
Q: also just interested why did you not use a JSON serialization?

Reply:
Please do know that stars are not a symptom of quality. We are already using this package inside the dynamic-sidecar so I did not want to introduce a new one.

Reply:
Yes, I think it's not just about quality; the other one also appears to be the official one for Python to work with msgpack. But it's up to you—I approved your PR.